fish_prompt.fish

```diff
@@ -846,7 +846,7 @@ set -g no_prompt_hist 'F'
 set -g symbols_style 'symbols'
 
 # Fix for WSL showing wrong command number at start
-if test (uname -r | /bin/grep [Mm]icrosoft)
+if test (uname -r | grep -i microsoft)

```

Comment:
was this originally about avoiding functions called `grep`? maybe use `command grep` to do the same thing?

Reply:
Good call!